user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementMetadataUploadDialog.java

```diff
@@ -828,12 +526,12 @@ public UploadItemsDisplay(Upload upload) {
       uploadSectionTitle.addClassName("section-title");
 
       var saveYourFileInfo = new InfoBox().setInfoText(
-              "Please save your excel file as UTF-16 Unicode Text (*.txt) before uploading.")
+              "Please save your Excel file as UTF-16 Unicode Text (*.txt) before uploading.")
```

Comment:
I would argue that this is no longer the only way as you can upload xlsx as well now.